src/main/java/com/bootcamp/demo/models/User.java

```diff
@@ -0,0 +1,72 @@
+package com.bootcamp.demo.models;
+
+import java.util.Objects;
+
+public final class User {
+    private final Long userId;
+    private String email;
```

Comment:
I think we will also need the location where the user is, to then look for the nearest scooters

Reply:
Hmm. If want to make a real-time experience yes, we need to know every time where is the user to quickly check the nearest scooters. If we don't want to do that, then we just need to use the location what is provided by the user when asks for nearest scooters. And we only updating the nearest scooters when the user asks explicitly again. What we will use then?